docs/src/keyboard_shortcuts.md

```diff
@@ -0,0 +1,31 @@
+Below is a list of supported commands and the corresponding keyboard shortcut (if assigned). Most of those are also available from the Command Palette, the main menu and the mouse context menu.
+
+Refer to the [Customization section](customization.md#keyboard-shortcuts-key-bindings) on how to modify or assign shortcuts to them.
+
+!!! Mac
+    If you using macOS, replace <kbd>ctrl</kbd> with <kbd>command</kbd>.
+
+| Feature | Shortcut | Command |
+| ------- | -------- | ------- |
+| Auto Complete | <kbd>ctrl</kbd> <kbd>space</kbd> (also on macOS) | `auto_complete`
+| Expand Selection | unbound | `lsp_expand_selection`
+| Find References | <kbd>shift</kbd> <kbd>f12</kbd> | `lsp_symbol_references`
```

Comment:
I just realized that this command overrides the default find reference command :)

Reply:
I didn't even know that ST provides one.

Reply:
There is no need to change anything now as this keybinding was here in a long time... 🙂

I just never was aware of it.